site/src/pages/TemplatePage/TemplateVersionsPage/VersionRow.tsx

```diff
@@ -32,6 +32,11 @@ export const VersionRow: React.FC<VersionRowProps> = ({
     onClick: () => navigate(version.name),
   });
 
+  const jobStatus = version.job.status;
+  if (jobStatus === "canceled") {
+    return null;
+  }
```

Comment:
I feel this filtering should be happening in the parent and not here 🤔

Reply:
after thinking about this a little more, I'm realizing this probably shouldn't be the frontends job at all. messes with pagination, etc. if we wanna filter them it should be on the backend.